From: Pavel Boldin (davinchi) <boldin.pavel@gmail.com>
Subject: Notify we've "found" a shell version so that the applet can register

Otherwise we may not have a secret agent when necessary to say, connect to
a VPN or a WPA-Enterprise network.

--- network-manager-applet-0.9.8.8.orig/src/shell-watcher.c	2013-11-11 16:41:44.000000000 +0200
+++ network-manager-applet-0.9.8.8/src/shell-watcher.c	2014-11-05 18:11:47.723591118 +0200
@@ -155,6 +155,8 @@
 
 	if (priv->retries++ == 5) {
 		g_warning ("Could not find ShellVersion property on org.gnome.Shell after 5 tries");
+		priv->shell_version = 0;
+		g_object_notify (G_OBJECT (watcher), "shell-version");
 		return;
 	}
 
